### ISSUE TITLE
Request for CCL to incorporate Eifler+ baryonic models

### ISSUE THREAD

**Issue by elisachisari** [LSSTDESC/CCL#236]
This was a request from the WL pipeline team. We should consider adding other models too.

**Comment by villarrealas** [comment]
Is this referring to his PCA analysis technique from https://arxiv.org/abs/1405.7423 ?

**Comment by elisachisari** [comment]
Yes, this is the correct reference. Maybe @timeifler and @creativeishu can comment.

**Comment by villarrealas** [comment]
My thoughts on how to go about this (could be me, or could be somebody else). Feel free to pipe in if this sounds terribly inefficient.

- We would need to store a file of {k, P(k)} ratios by @timeifler or allow input of such a file at runtime.
- A flag for the cosmology object should be aware of this option being turned on or off.
- Upon running ccl_nonlin_matter_power or similar, it should check for this option before returning P(k). If this option has been flagged, it should apply the ratio to the output power spectrum.
- Since we are allowing the return of an arbitrary P(k), we would need to do some sort of spline interpolation of the ratios **OR** not allow it to be utilized outside of the boundaries of our data.

I believe this is all we require. I am inclined to say that we should have it as part of an input configuration file, but I do not think we currently have anything that functions in this way. I do not believe additional changes need to be made after the P(k) transformation is applied.

**Comment by c-d-leonard** [comment]
@elisachisari do we know if this is still something that is wanted in terms of baryonic corrections? or is something else a priority at this point?
